terraform-build-user/main.tf

```diff
@@ -9,6 +9,9 @@ module "iam_user" {
     aws.images-staging-ssm    = aws.images-staging-ssm
   }
 
-  ssm_parameters = ["/windows/commando/administrator/password"]
-  user_name      = "build-windows-commando-vm-packer"
+  # This image can take a while to build, so we set the max session
+  # duration to 3 hours.
+  ec2amicreate_role_max_session_duration = 3 * 60 * 60
```

Comment:
Even though you set the max session duration for the build role to 3 hours here, the build role is only assumed for 2 hours in the GitHub build action:
https://github.com/cisagov/windows-commando-vm-packer/blob/e428dbbc8a8f6394b3650fc672d0bdca3316cc41/.github/workflows/build.yml#L205-L212

Is this intentional or should we keep these two values in sync with one another?

Reply:
@dav3r resolved with [8e578e1](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/8e578e130bb2f44b9f850f481980f3d420a0c232)